howto/application/extend.md

```diff
@@ -46,17 +53,124 @@ The following example shows how to use application hooks to change the Android s
    sleep 5
    ```
 
-1. Make the hooks executable:
+   [note type="information" status="Important"]
+   In the file, replace `<hook_dir>` with the [environment variable](https://discourse.ubuntu.com/t/hooks/28555#env-variables) that points to the current working directory for the hook. If you plan to run the hook [as an application hook](#application-hooks), use `$APP_DIR`. If you plan to run the hook [through an addon](#addon), use `$ADDON_DIR`.
```

Comment:
Strictly speaking, this is not accurate and introduces confusion to me.
If calling it `<hook_dir>` (depends on where people put the apk file), I thought the apk is placed into
 1. <ADDON_DIR>/hooks/ 
or
 2. <APP_DIR>/hooks/ 

However in this note section, we are saying 
```
replace `<hook_dir>` with <ADDON_DIR> or <APP_DIR>
```
So if people put the apk under either 1) or 2), things won't work. 

Maybe consider using `<app_or_addon_dir>` over `<hooks_dir>`

Reply:
How about `<runtime_dir>` or `<home_dir>`?

Reply:
Or `<working_dir>`, since that is how we describe it (the current working directory for the hook)?

Reply:
+1. The `<working_dir>` fits better than others here